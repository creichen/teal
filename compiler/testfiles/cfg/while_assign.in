fun f(a) = {
  while a {
    a := 0;
  }
}

