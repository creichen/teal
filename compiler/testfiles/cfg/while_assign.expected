Entry#0 @ 1 :
  -> Access#0
VarDecl#0 @ 1 :
Access#0 @ 2 :
  -> IntConstant#0
  -> Exit#0
Access#1 @ 3 :
  -> AssignStmt#0
AssignStmt#0 @ 3 :
  -> Access#0
IntConstant#0 @ 3 :
  -> Access#1
Exit#0 @ 5 :
